_posts/blog/2021-04-08-whitakerlab-at-collabw21.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: The WhitakerLab at CollabW21
+categories: blog
+excerpt: Notes from the Lab's feedback about the SSI organised event
+tags: [lab-meeting] ### INSERT TAGS IF APPLICABLE
+image:
+  feature:
+link:
+date: 2021-04-08 ### UPDATE date
+modified:
+share: true
+author: sarah_gibson ### CHANGE to your author name (in _data/authors.yml)
+---
+
+At the end of March 2021, many members of the Whitaker Lab attended the Software Sustainability Institute's Collaborations Workshop.
+At the following lab meeting, those who attended shared their experiences with the rest of the lab.
+This is a blog post covering the event and the lab member's reflections from it.
+
+## What is the SSI?
+
+The [Software Sustainability Institute](https://software.ac.uk/about) (SSI for short) is a network of UK universities, Research Software Engineering groups and policy makers dedicated in improving the quality, sustainability and recognition of research software.
+They help the members in their network learn software skills and best practices, and advocate for culture change in their organisations and institutions.
+
+Many members of the lab (current and past) are also [SSI Fellows](https://software.ac.uk/programmes-and-events/fellowship-programme).
+[Can you spot them?](https://software.ac.uk/about/fellows)
+
+## What is Collaborations Workshop?
+
+The [Collaborations Workshop](https://software.ac.uk/programmes-and-events/collaborations-workshops) (often shorted to CollabW) is an annual, 3-day event focused around bringing together researchers, software developers, data librarians, managers, funders and anyone else who cares about research software to foster interdisciplinary collaborations and explore important ideas around software and research.
+
+CollabW features a few keynotes but it is largely run as an [unconference event](https://unconference.net/unconferencing-how-to-prepare-to-attend-an-unconference/) with lightning talks, discussion sessions, mini workshops, and speed blogging events all relating to one or a few central themes.
+This feels radically different to traditional conference events as you are elevated from audience member to participant, giving you the opportunity to create something during the event and have discussions with a wide variety of other attendees.
+
+The final day features a [hack day](https://software.ac.uk/cw21/hack-day) where small teams can work on community-generated projects addressing a need they see in the research software community.
```

Comment:
Would you be able to give just a few examples of the community-generated projects here?